AISKU/package.json

```diff
@@ -15,11 +15,11 @@
         "registry": "https://mseng.pkgs.visualstudio.com/_packaging/ApplicationInsights-Team/npm/registry"
     },
     "dependencies": {
-        "applicationinsights-analytics-js": "0.0.x",
-        "applicationinsights-channel-js": "0.0.x",
-        "applicationinsights-common": "0.0.x",
-        "applicationinsights-core-js": "0.0.x",
-        "applicationinsights-properties-js": "0.0.x",
+        "applicationinsights-analytics-js": "0.0.89",
+        "applicationinsights-channel-js": "0.0.31",
+        "applicationinsights-common": "0.0.77",
+        "applicationinsights-core-js": "0.0.79",
+        "applicationinsights-properties-js": "0.0.4",
```

Comment:
update to odd and update this package (to use updated core/common) then udpate this one.